packages/taquito-signer/src/mnemonicHelpers.ts

```diff
@@ -0,0 +1,72 @@
+import { hmac } from "@stablelib/hmac";
+import { SHA512 } from "@stablelib/sha512"
+import { generateKeyPairFromSeed } from "@stablelib/ed25519";
+import { b58cencode, prefix } from "@taquito/utils";
+import { InvalidDerivationPathError } from "./mnemonicDerivationErrors";
+
+export interface KeyNode {
+  privateKey: Uint8Array;
+  chainCode: Uint8Array;
+}
+export const createNode = (seed: Uint8Array, key: Uint8Array): KeyNode => {
+  const derived = hmac(SHA512, key, seed);
+
+  return {
+    privateKey: derived.slice(0, 32),
+    chainCode: derived.slice(32)
+  };
+}
+
```

Comment:
I'm noticing some missing semi-colons and extra white spaces like these ones, could probably fix it by running the linter through it